src/webui/src/components/PackageSidebar/modules/LastSync/index.jsx

```diff
@@ -19,14 +19,18 @@ export default class LastSync extends React.Component {
       }
     });
 
-    return lastUpdate ? datetime(lastUpdate) : '';
+    const datetime = format(new Date(lastUpdate), 'MMM D, YYYY, h:mm A');
+
+    return lastUpdate ? datetime : '';
   }
 
   get recentReleases() {
     let recentReleases = Object.keys(this.props.packageMeta.time).map((version) => {
+      const datetime = format(new Date(this.props.packageMeta.time[version]),
+        'MMM D, YYYY, h:mm A');
       return {
         version,
-        time: datetime(this.props.packageMeta.time[version])
+        time: datetime
```

Comment:
use `time` instead `datetime` and then return `{version, time}`

Reply:
`let` or `const`?
Which one should be used in this project? @juanpicado 

---

Also you can use shorthand for `time: datetime` by change variable name to `time`

Reply:
Yeah, that's I meant

Reply:
@Meeeeow rule for me is simple, `const` whether the variable won't change. Something else `let`.